Build error with custom Webrender

Trying to build Servo with custom Webrender and Webrender_Api results in 2 build errors. 

```
error[E0432]: unresolved import `webrender_api::ScrollEventPhase`
  --> components/compositing/compositor.rs:39:37
   |
39 | use webrender_api::{LayoutVector2D, ScrollEventPhase, ScrollLocation};
   |                                     ^^^^^^^^^^^^^^^^ no `ScrollEventPhase` in the root

error[E0204]: the trait `Copy` may not be implemented for this type
   --> components/compositing/compositor.rs:193:17
    |
193 | #[derive(Clone, Copy)]
    |                 ^^^^
...
202 |     phase: ScrollEventPhase,
    |     ----------------------- this field does not implement `Copy`

error: aborting due to 2 previous errors

```

Looks like `ScrollEventPhase` was removed in https://github.com/servo/webrender/pull/2629.

--- a/servo-tidy.toml
+++ b/servo-tidy.toml
@@ -39,13 +39,9 @@ num = [
 # Ignored packages with duplicated versions
 packages = [
   "bitflags",
-  "itoa",
   "log",
   "num-traits",
-  "quote",
   "rand",
-  "syn",
-  "unicode-xid",
   "winapi",
 ]
 # Files that are ignored for all tidy and lint checks.

